cfgrammar/src/lib/mod.rs

```diff
@@ -4,6 +4,7 @@
 #![allow(clippy::unnecessary_wraps)]
 #![allow(clippy::upper_case_acronyms)]
 #![forbid(unsafe_code)]
+#![deny(unreachable_pub)]
```

Comment:
perhaps these should use `#![deny(rustc::unreachable_pub)]` in accordance with the `clippy::lint_name` usage above?

Reply:
I have no idea! I'm fine either way, so let's do whichever you feel most comfortable with.

Reply:
I had a look through https://doc.rust-lang.org/rustc/lints/index.html and I couldn't find using the `rustc::` prefix actually documented anywhere, so I'm kind of inclined to leave it as is.